### DISCUSSION TITLE
data cooperative

### DISCUSSION THREAD

**Discussion by Cancel-Rent** [trinodb/trino#17806] [Q&A]
Hey all excited to have found you all. Looking to have some awesome person with data-niche knowledge & dreams concerning _our_
 (collective society) 
data-futures and would be interested in in talking about it with some artists for a multi-media documentary piece about a 100+ member artist based tenants union currently on rent strike against a notorious group of landlords in Bushwick, Brooklyn, NY? 
👾🧡👾 
my name is shannon lumpkin, a member and producer. 15108007115 or hey.neighbor@ohtu.space

**Comment by martint** [comment]
Closing as off-topic.
